.github/workflows/release.yaml

```diff
@@ -166,18 +166,40 @@ jobs:
       - run: npm install -g vsce typescript@4.7.4 esbuild
       - run: cp CHANGELOG.md LICENSE vscode-ext/
       - run: cp /tmp/artifacts/x86_64-unknown-linux-gnu/libmirrord_layer.so vscode-ext/
-      - run: cd vscode-ext && npm install && tsc && vsce publish ${{ steps.version.outputs.version }} --pre-release --target linux-x64
+      - run: mkdir -p /tmp/vscode-ext
+      - run: |
+          cd vscode-ext 
+          npm install          
+          vsce package -o /tmp/vscode-ext/vscode-ext-linux-x64 --target linux-x64
```

Comment:
Don't you need to run `tsc` before `vsce package`?

Reply:
oops my bad, i didnt publish the commit